forms-web-app/src/services/task.service.js

```diff
@@ -118,8 +125,9 @@ const SECTIONS = {
       href: `/${VIEW.APPELLANT_SUBMISSION.SITE_LOCATION}`,
       rule: statusAppealSiteAddress,
     },
```

Comment:
I forgot to mention that what you could also do is update the checkYourAnswer rule to activate the link only when you new health task is done.  I can do it in my PR if you want

Reply:
It's basically just adding your rule in the list, but I work around that at the moment so I can do it if you want to merge and focus on 830

Reply:
yeah - I'm a bit unsure on this, if you want to pick it up that would be helpful. Thanks